modules/new_serverpod_auth/serverpod_auth_migration/serverpod_auth_migration_server/test/integration/auth_migration_email_test.dart

```diff
@@ -125,6 +129,59 @@ void main() {
         },
       );
 
+      test(
+        'when running `migrateWithoutPassword` with a throwing `userMigrationHook`, then it forwards the error and does not create an `AuthUser`.',
+        () async {
+          AuthMigrationEmail.config = AuthMigrationConfig(
```

Comment:
Could `AuthMigrationEmail.config = ...` be extracted out in a `setUp` since all three tests share the same?

Reply:
Ahh sorry yes there's only 2 tests, my bad. It's not bad right now, but it would read slightly better to have the throwing config shared in a group. Correct me if I am wrong, but I don't think you'd need an entire `withServerpod`, no? Wouldn't a group with a `setUp` and `tearDown` be enough?

Reply:
I understand :) Your changes are great with the `withServerpod` :D